gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -67,23 +94,52 @@ def call_quit(self, _):
         return True
 
     def call_load(self, other_args):
+        """Load command - get df from gecko"""
         self.current_coin, self.current_df = pycoingecko_view.load(other_args)
 
     def call_view(self, other_args):
+        """View command - plot loaded coin"""
         if self.current_coin:
             pycoingecko_view.view(self.current_coin, self.current_df, other_args)
 
         else:
             print("No coin selected. Use 'load' to load the coin you want to look at.")
             print("")
 
+    def call_trend(self, _):
+        """Trend command - show top 7 gecko coins"""
+        pycoingecko_view.trend()
+
     def call_top(self, other_args):
+        """Top command - get top n coins from coinmarketcap"""
         cmc_view.get_cmc_top_n(other_args)
 
+    def call_book(self, other_args):
+        """Book command - get order book from binance"""
+        binance_model.order_book(self.current_coin + self.current_currency, other_args)
+
+    def call_candle(self, other_args):
+        """Candle command - show candle chart from binance"""
+        binance_model.show_candles(
+            self.current_coin + self.current_currency, other_args
+        )
+
+    def call_balance(self, _):
+        """Balance command - check current holdings of coin in binance"""
+        binance_model.balance(self.current_coin)
+
+    def call_add(self, other_args):
+        """Add command - set current coin for binance operations"""
+        self.current_coin, self.current_currency = binance_model.add_binance_coin(
+            other_args
+        )
+        print(f"{self.current_coin} loaded vs {self.current_currency}")
```

Comment:
I think this should be done in the add_binance_coin and not controller. The controller should be just a middleman let's say.

Also, if it fails loading, it shouldn't print anything. Otherwise we get something like:
```
None loaded vs None
```